morebits.css

```diff
@@ -40,6 +40,7 @@ form.quickform
 form.quickform *
 {
 	font-family: sans-serif;
+	font-size: 12px;
```

Comment:
Why this and not removing `font-size` from `.ui-dialog.morebits-dialog .morebits-dialog-content`? https://github.com/azatoth/twinkle/blob/07fffdf04bf8adf008862eb2ac2e5cc52c28b9d9/morebits.css#L224

Reply:
I was thinking of this more as a proposal so wasn't paying attention to the code. Let's see. Good old days of playing with the CSS...